pints/tests/test_mcmc_controller.py

```diff
@@ -1838,6 +1838,121 @@ def test_multi_chain_with_nones(self):
         self.assertTrue(np.all(log_pdfs1 == log_pdfs2))
 
 
+class TestMCMCInitialisationMethod(unittest.TestCase):
```

Comment:
Can we stick this in its own file? I wouldn't know to look for it here if I needed it

Reply:
Done